engine/lite/k8sDeploymentManager/src/test/scala/pl/touk/nussknacker/k8s/manager/KafkaK8sSupport.scala

```diff
@@ -0,0 +1,108 @@
+package pl.touk.nussknacker.k8s.manager
+
+import akka.Done
+import akka.stream.scaladsl.{Sink, Source}
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest.Matchers
+import pl.touk.nussknacker.k8s.manager.KafkaK8sSupport.kafkaService
+import pl.touk.nussknacker.test.VeryPatientScalaFutures
+import skuber.Container.Running
+import skuber.api.client.KubernetesClient
+import skuber.json.format._
+import skuber.{Container, EnvVar, ObjectMeta, Pod, Service}
+
+import scala.concurrent.{ExecutionContext, Future, Promise}
+
+object KafkaK8sSupport {
+
+  val kafkaService = "kafka-k8s-test"
+
+}
+
+//TODO: would it be faster if we run e.g. kcat as k8s job instead of exec into kafka pod?
+class KafkaK8sSupport(k8s: KubernetesClient) extends VeryPatientScalaFutures with LazyLogging with Matchers {
+
+  //set to false in development to reuse existing kafka pod
+  private val cleanupKafka = true
+
+  private val kafkaPod = "kafka-k8s-test"
+
+  def start()(implicit ec: ExecutionContext): Unit = if (k8s.getOption[Pod](kafkaPod).futureValue.isEmpty) {
+    val kafkaContainer = Container(
+      name = kafkaPod,
+      image = "debezium/kafka:1.8",
```

Comment:
Can u add a notice why u used debezium and why 1.8? Looks like outdated image

Reply:
Added comment, 1.8 is the latest on dockerhub

Reply:
Ok, so they don't use kafka versioning? or it is based on kafka 1.8? thanks for comment

Reply:
no, they have their own way